lib/page_steps.js

```diff
@@ -71,3 +71,249 @@ step(
     assert.strictEqual(valMatches[idx - 1] === val, true);
   }
 );
+
+step("Click <text> in spec memory", async (text) => {
+  const propertyValue = gauge.dataStore.specStore.get(text);
+  await taiko.click(propertyValue);
+});
+
+step("Write <text> from property into <textbox>", async (text, textbox) => {
+  const propertyValue = process.env[text];
+  await taiko.write(propertyValue, taiko.into(taiko.textBox(textbox)));
+});
+
+step("Write <text> from spec memory into <textbox>", async (text, textbox) => {
+  const propertyValue = gauge.dataStore.specStore.get(text);
+  await taiko.write(propertyValue, taiko.into(taiko.textBox(textbox)));
+});
+
+step("wait <milliseconds> milliseconds", async (milliseconds) => {
```

Comment:
To be consistent I propose we start the step with uppercase

Reply:
That's a good suggestion @ramsbaron. Each step is read like a sentence and capitalizing it makes it easy to read.